include/fastdds/rtps/security/logging/Logging.h

```diff
@@ -105,6 +108,10 @@ class Logging
    */
   LoggerListener const* get_listener() const { return listener_; }
 
+  bool set_guid(const GUID_t& guid, SecurityException& exception);
+
+  bool set_domaine_id(const uint32_t id, SecurityException& exception);
```

Comment:
```suggestion
  bool set_domain_id(const uint32_t id, SecurityException& exception);
```

Reply:
Oups. Will fix it everywhere in a separate commit.